packages/ods/src/components/range/tests/behavior/ods-range.e2e.ts

```diff
@@ -0,0 +1,230 @@
+import type { E2EElement, E2EPage } from '@stencil/core/testing';
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('ods-range behavior', () => {
```

Comment:
we went with the `behaviour` style for now, please update the directory name accordingly